extensions/wikia/Email/templates/main.mustache

```diff
@@ -42,44 +42,45 @@
+								{{# marketingFooter}}{{# icons.Comics}}<img src="{{url}}" width="{{width}}" height="{{height}}" class="hub-icon">{{/ icons.Comics}}{{/ marketingFooter}}
 								{{hubsMessages.comics}}
 							</a>
 							<a href="{{hubsMessages.videoGamesURL}}">
-								{{# marketingFooter}}<img src="http://vignette1.wikia.nocookie.net/wikianewsletter/images/e/e8/Games.gif/revision/latest?cb=20150330185144" class="hub-icon">{{/ marketingFooter}}
+								{{# marketingFooter}}{{# icons.Games}}<img src="{{url}}" width="{{width}}" height="{{height}}" class="hub-icon">{{/ icons.Games}}{{/ marketingFooter}}
 								{{hubsMessages.videoGames}}
 							</a>
 							<a href="{{hubsMessages.moviesURL}}">
-								{{# marketingFooter}}<img src="http://vignette1.wikia.nocookie.net/wikianewsletter/images/b/bf/Movies.gif/revision/latest?cb=20150330185202" class="hub-icon">{{/ marketingFooter}}
+								{{# marketingFooter}}{{# icons.Movies}}<img src="{{url}}" width="{{width}}" height="{{height}}" class="hub-icon">{{/ icons.Movies}}{{/ marketingFooter}}
 								{{hubsMessages.movies}}
 							</a>
 							<a href="{{hubsMessages.lifestyleURL}}">
-								{{# marketingFooter}}<img src="http://vignette4.wikia.nocookie.net/wikianewsletter/images/6/60/Lifestyle.gif/revision/latest?cb=20150330185152" class="hub-icon">{{/ marketingFooter}}
+								{{# marketingFooter}}{{# icons.Lifestyle}}<img src="{{url}}" width="{{width}}" height="{{height}}" class="hub-icon">{{/ icons.Lifestyle}}{{/ marketingFooter}}
 								{{hubsMessages.lifestyle}}
 							</a>
 							<a href="{{hubsMessages.musicURL}}">
-								{{# marketingFooter}}<img src="http://vignette4.wikia.nocookie.net/wikianewsletter/images/f/fb/Music.gif/revision/latest?cb=20150330185211" class="hub-icon">{{/ marketingFooter}}
+								{{# marketingFooter}}{{# icons.Music}}<img src="{{url}}" width="{{width}}" height="{{height}}" class="hub-icon">{{/ icons.Music}}{{/ marketingFooter}}
 								{{hubsMessages.music}}
 							</a>
 							<a href="{{hubsMessages.booksURL}}">
-								{{# marketingFooter}}<img src="http://vignette4.wikia.nocookie.net/wikianewsletter/images/c/c2/Books.gif/revision/latest?cb=20150330185117" class="hub-icon">{{/ marketingFooter}}
+								{{# marketingFooter}}{{# icons.Books}}<img src="{{url}}" width="{{width}}" height="{{height}}" class="hub-icon">{{/ icons.Books}}{{/ marketingFooter}}
 								{{hubsMessages.books}}
 							</a>
 							<a href="{{hubsMessages.tvURL}}">
-								{{# marketingFooter}}<img src="http://vignette1.wikia.nocookie.net/wikianewsletter/images/c/c5/TV.gif/revision/latest?cb=20150330185219" class="hub-icon">{{/ marketingFooter}}
+								{{# marketingFooter}}{{# icons.TV}}<img src="{{url}}" width="{{width}}" height="{{height}}" class="hub-icon">{{/ icons.TV}}{{/ marketingFooter}}
 								{{hubsMessages.tv}}
 							</a>
```

Comment:
I have a feeling Front End would prefer things like this stay expanded so they more closely represent the HTML output rather than code.  @lizlux (since you worked on this), @bkoval, thoughts on this?